### ISSUE TITLE
prompt

### ISSUE THREAD

**Issue by lmr2706** [uncbiag/SegNext#12]
Hello, I would like to know how you input multiple types of prompts. For example, are the point, box, and scribble prompts automatically generated in the model based on the error regions? What about in the testing stage? Moreover, are these multiple prompts simply added together, or is there a specific mechanism that enables them to work collaboratively and complement each other?

**Comment by lmr2706** [comment]
Additionally, I would like to know if it is necessary to download the over 100,000 images from the LVIS training set for training purposes.

**Comment by qinliuliuqin** [comment]
@lmr2706 Hi, thank you for your thought-provoking questions.
Q1. Are the point, box, and scribble prompts automatically generated in the model based on the error regions?
No. We directly use the ground truth mask to simulate visual prompts such as boxes, scribbles, and polygons. We only evaluate one-round interaction performance, so we don't use the error regions to simulate the next-round interactions. 

Q2. What about in the testing stage?
First of all, we do not train our model with diverse prompts. As we mentioned in the caption of Table 3, our method is not trained with diverse prompts, it generalizes reasonably well on different “unseen” visual prompts. The testing stage is simply the process described in Q1.

Q3. Are these multiple prompts simply added together, or is there a specific mechanism that enables them to work collaboratively and complement each other?
No. We don't combine multiple prompts together for evaluation. We evaluate each prompt separately with only one round. You may refer to [nnInteractive](https://github.com/MIC-DKFZ/nnInteractive) on how to enable diverse prompts work collaboratively. The basic idea is to encode different prompts in different feature channels. Our work provides a proof-of-concept that dense maps can naturally support diverse visual prompts. However, further investigation on to collaboratively use them to further boost the performance is beyond the scope of our work.

Q4. Is is necessary to download the over 100,000 images from the LVIS training set for training purposes?
Yes, LVIS provides high-quality segmentation masks, higher than the SA-1B. These high-quality segmentation masks are the key to train a good interactive segmentation model. LVIS share the same image set with COCO, so you only need to download the masks.
